api/utils/json.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 Gravitational, Inc.
+Copyright 2021 Gravitational, Inc.
```

Comment:
Don't really want to be nitpicking on the license headers, but I think we usually append the last year, not replace it so `2015-2021`. Noticed the same somewhere above as well.

Reply:
Didn't know this, thanks!